src_py/apiServer/receiver.py

```diff
@@ -36,13 +36,13 @@ def initReceiver(receiverHost, receiverPort, event):
 
 def processResult(resData, currentPhase):
         if (currentPhase == "Training"):
-            # Parse the Result:
+            # Parse the Result: [w#, float]
             worker = resData[0]
             result = float(resData[1].replace(' ',''))
             #print(result)
-            if (int(result) == -1):         # TODO what is this??
+            if (int(result) == -1):
                 print(f"Received loss=-1 from worker {worker}. The NN's weights have been reset.")
-            if (int(result) != -1 and int(result != 0)):
+            if (int(result) != -1 and int(result != 0)):    # weird condition
```

Comment:
What are the number -1 and 0? 
Add a comment that explain it or use "definitions" such as 

DEFINITION_A = -1 
DEFINITION_B = 0

Reply:
Done